utensor_cgen/__main__.py

```diff
@@ -71,6 +72,9 @@ def _build_parser():
   parser.add_argument("--debug-comment", dest="debug_cmt",
                       action="store_true",
                       help="Add debug comments in the output source file (default: %(default)s)")
+  parser.add_argument("-Q", "--quantize", dest="quantize_flag",
+                      action="store_true", default=False,
```

Comment:
If `action` is `store_true`, the default value is automatically `False`.
The `default` keyword argument is not necessary here.
I prefer to remove it.

Reply:
But if you like it this way, I'm good.